content/en/blog/2022/dotnet-instrumentation-first-beta.md

```diff
@@ -0,0 +1,42 @@
+---
+title: OpenTelemetry .NET Automatic Instrumentation Releases its first Beta
+date: 2022-05-12
+author: OpenTelemetry .NET Automatic Instrumentation Team
+---
+
+The .NET community already has a stable OpenTelemetry SDK. Using the SDK requires engineers 
+to modify and rebuild their applications, which may be undesirable or not possible in some 
+scenarios. Furthermore, the OpenTelemetry SDK does not provide instrumentation for all libraries 
+or versions of libraries, and as a result, there is a desire to leverage 
+[byte-code instrumentation](https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/blob/v0.1.0-beta.1/docs/design.md#bytecode-instrumentations) 
+to support gathering telemetry from those libraries. The 
+[OpenTelemetry .NET Automatic Instrumentation](https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation) 
+project strives to solve both of those problems.
+
+The [first beta release](https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/releases/tag/v0.1.0-beta.1) 
+of this project includes support for:
+
+* Gathering trace data from .NET applications without requiring code changes[^devopsnote]
+* Gathering trace data from .NET libraries that the SDK does not support[^librarysupport]
+
+See the [examples](https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/tree/v0.1.0-beta.1/examples) 
+for demonstrations of different instrumentation scenarios covered by the OpenTelemetry .NET 
+Automatic Instrumentation.
+
+Over the next few months we plan to:
+
+* Support additional instrumentation libraries
```

Comment:
Is there a good link to put here so people can see which libraries are supported by the agent today? I think that will be one of the first questions on peoples' minds.

Reply:
Yes this would be the link to the markdown in the tagged release: https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/blob/v0.1.0-beta.1/docs/config.md#instrumented-libraries-and-frameworks